doc/api/net.md

```diff
@@ -246,168 +352,217 @@ server.on('error', (e) => {
 });
 ```
 
-(Note: All sockets in Node.js are set `SO_REUSEADDR`.)
+*Note*: All `net.Socket` objects in Node.js are set to use the `SO_REUSEADDR`
+flag by default.
 
 ### server.listening
 <!-- YAML
 added: v5.7.0
 -->
 
-A Boolean indicating whether or not the server is listening for
-connections.
+A boolean indicating whether or not the server is listening for connections.
 
 ### server.maxConnections
 <!-- YAML
 added: v0.2.0
 -->
 
-Set this property to reject connections when the server's connection count gets
-high.
+The `server.maxConnections` property can be set to specify the maximum number
+of connections the server will accept. Once the current number of connections
+reaches this threshold, all additional connection requests will be rejected.
+The default is `undefined`.
 
-It is not recommended to use this option once a socket has been sent to a child
-with [`child_process.fork()`][].
+*Note*: Setting this property *after* a socket has been sent to a child
+using [`child_process.fork()`][] is *not recommended*.
 
 ### server.ref()
 <!-- YAML
 added: v0.9.1
 -->
 
-Opposite of `unref`, calling `ref` on a previously `unref`d server will *not*
-let the program exit if it's the only server left (the default behavior). If
-the server is `ref`d calling `ref` again will have no effect.
+Calling the `server.ref()` method will cause the Node.js event loop to continue
+so long as the `server` is not closed. Multiple calls to `server.ref()` will
+have no effect.
```

Comment:
It looks like you are  trying to get out of the double-negative situation of previous docs, but the new text implies that ref() must be called, when sockets are refed by default.

I feel quite strongly that the docs for the event loop and aliveness should be in one place, and this should just link, as should the many other resources.

At least, the text should be consistent between the various handle-based resources, like https://nodejs.org/api/timers.html#timers_timeout_ref